packages/frontend/components/composed/journal/CurrentIssue/CurrentIssue.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { graphql } from "react-relay";
+import { useTranslation } from "react-i18next";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import * as Styled from "./CurrentIssue.styles";
+import { CoverImage, ArrowLink, PrecisionDate } from "components/atomic";
+import ContributorsList from "components/composed/contributor/ContributorsList";
+import { CurrentIssueFragment$key } from "@/relay/CurrentIssueFragment.graphql";
+
+export default function CurrentIssue({ data }: Props) {
+  const { t } = useTranslation();
+  const result = useMaybeFragment(fragment, data);
+  const issue = result?.edges[0].node;
```

Comment:
If there's no edges, then the frontend throws an error: `TypeError: Cannot read properties of undefined (reading 'node')`
https://wdp-frontend-a4e1q0zaf-cast-iron.vercel.app/collections/LvMzrv0hnkPTxevKFkekgHpXVR5y1bJ